### PR TITLE
[wasm][aot] Disable tests that are failing because they depend on stacktrace API

### PR DESCRIPTION
**Pull request by radical**
- And update issue urls, as we have merged the issues into one tracking issue (https://github.com/dotnet/runtime/issues/50957)

### CODE REVIEW

File: src/libraries/tests.proj [hunk 1]
```diff
@@ -291,7 +291,7 @@
     <!-- Issue: https://github.com/dotnet/runtime/issues/51961 -->
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Text.Json/tests/System.Text.Json.Tests/System.Text.Json.Tests.csproj" />
 
-    <!-- Issue: https://github.com/dotnet/runtime/issues/50959 -->
+    <!-- Issue: https://github.com/dotnet/runtime/issues/50957 -->
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.IO.Pipelines\tests\System.IO.Pipelines.Tests.csproj" />
```
Comment (lewing) [inline src/libraries/tests.proj:295]: can't we remove the exclusions?

Comment (radical) [reply src/libraries/tests.proj:295]: Yep, this and more. But as a separate PR.
This one will unblock the tracking PR. And then a follow up PR would enable the ones that have been stable on the tracking one.

**Review by lewing — APPROVED** [review]

**Pull request merged**